draft-terbu-sd-jwt-vc.md

```diff
@@ -1,22 +1,20 @@
 %%%
-title = "SD-JWT-based Verifiable Credentials with JSON payloads (SD-JWT VC)"
-abbrev = "SD-JWT VC"
-ipr = "trust200902"
-workgroup = "TODO Working Group"
-keyword = ["security", "openid", "sd-jwt"]
+title = "Native JWT Representation of Verifiable Credentials"
```

Comment:
```suggestion
title = "JWT representation of Verifiable Credentials"
```

I don't like "native". What does this mean? I would rather like to have a title along the lines of "Verifiable Credentials with JSON payload following JWT rules with optional selective disclosure support" but I know that sounds pretty artificial. So removing native might be ok.